script/tool/lib/src/version_check_command.dart

```diff
@@ -358,11 +359,19 @@ ${indentation}HTTP response: ${pubVersionFinderResponse.httpResponse.body}
       }
     }
 
-    final Version? fromChangeLog =
-        versionString == null ? null : Version.parse(versionString);
-    if (fromChangeLog == null) {
-      printError(
-          '${indentation}Cannot find version on the first line CHANGELOG.md');
+    if (versionString == null) {
+      printError('${indentation}Unable to find a version in CHANGELOG.md');
+      print('${indentation}The current version should be on a line starting '
+          ' with "## ", either on the first non-empty line or after a "## NEXT" '
```

Comment:
```suggestion
          'with "## ", either on the first non-empty line or after a "## NEXT" '
```

This was printing `starting  with` (double space), and breaking the test.

Reply:
Thanks; I was being lazy and not re-syncing and fixing locally. If this fails again I'll actually re-run locally...